lib/config/decrypt.ts

```diff
@@ -209,3 +228,29 @@ Refer to migration documents here: https://docs.renovatebot.com/mend-hosted/migr
   logger.trace({ config: decryptedConfig }, 'decryptedConfig');
   return decryptedConfig;
 }
+
+function getAzureCollection(): string | undefined {
+  const platform = GlobalConfig.get('platform');
+  if (platform !== 'azure') {
+    return undefined;
+  }
+
+  const endpoint = GlobalConfig.get('endpoint');
+  const endpointUrl = parseUrl(endpoint);
+  if (endpointUrl === null) {
+    // should not happen
+    logger.warn({ endpoint }, 'Unable to parse endpoint for token decryption');
+    return undefined;
+  }
+
+  const azureCollection = endpointUrl.pathname
+    ?.split('/')
+    .filter(is.nonEmptyString)
+    .join('/');
+  if (!is.nonEmptyString(azureCollection)) {
+    logger.warn({ endpoint }, 'Unable to find azure collection name from URL');
```

Comment:
we should not warn for self hosted devops server. they don't have azure collections

Reply:
downgraded to debug